scripts/skip-upgrade-tests.txt

```diff
@@ -165,4 +165,6 @@ vcd.TestAccVcdVAppVmDhcpWait.tf v3.0.0 "Terraform 0.13+ incompatibility"
 vcd.TestVappVmDS.tf v3.0.0 "Terraform 0.13+ incompatibility"
 vcd.TestAccVcdVappNetworkDS.tf v3.0.0 "Terraform 0.13+ incompatibility"
 vcd.TestAccVcdVappOrgNetworkDS.tf v3.0.0 "Terraform 0.13+ incompatibility"
-vcd.TestAccVcdEdgeGatewaySettingsFull.tf v3.0.0 "Terraform 0.13+ shows expected diff for 'output', but it cannot be ignored"
\ No newline at end of file
+vcd.TestAccVcdEdgeGatewaySettingsFull.tf v3.0.0 "Terraform 0.13+ shows expected diff for 'output', but it cannot be ignored"
```

Comment:
Reason (Terraform 0.14 started showing diffs for `output` and we are getting these):
```
Changes to Outputs:
  ~ resources = [
        # (16 unchanged elements hidden)
        "vcd_network_isolated",
      + "vcd_network_isolated_v2",
        "vcd_network_routed",
      + "vcd_network_routed_v2",                                                                                                                               "vcd_nsxt_edgegateway",
      + "vcd_nsxt_network_dhcp",
      + "vcd_nsxt_network_imported",
        "vcd_nsxv_dhcp_relay",
        # (15 unchanged elements hidden)
        "vcd_vapp_vm",
      + "vcd_vm",
        "vcd_vm_affinity_rule",
        # (2 unchanged elements hidden)
    ]


```